resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/kubernetes/SparkPodInitContainerBootstrapSuite.scala

```diff
@@ -27,6 +27,7 @@ import org.apache.spark.deploy.kubernetes.constants._
 class SparkPodInitContainerBootstrapSuite extends SparkFunSuite with BeforeAndAfter {
   private val OBJECT_MAPPER = new ObjectMapper()
   private val INIT_CONTAINER_IMAGE = "spark-init:latest"
+  private val DOCKER_IMAGE_PULL_POLICY = "IfNotPresent"
```

Comment:
We could if we don't foresee the values ever change. Validating here gives us the burden to maintain this list up to date with kubernetes, so if we're open to do this for this and other option moving forward I'm open to it.